dagger/run_dbt_spark_tests.py

```diff
@@ -0,0 +1,139 @@
+import os
+
+import argparse
+import sys
+
+import anyio as anyio
+import dagger as dagger
+from dotenv import find_dotenv, load_dotenv
+
+PG_PORT = 5432
+load_dotenv(find_dotenv("test.env"))
+DEFAULT_ENV_VARS = {
+    "DBT_TEST_USER_1": os.getenv("DBT_TEST_USER_1", "buildbot+dbt_test_user_1@dbtlabs.com"),
+    "DBT_TEST_USER_2": os.getenv("DBT_TEST_USER_2", "buildbot+dbt_test_user_2@dbtlabs.com"),
+    "DBT_TEST_USER_3": os.getenv("DBT_TEST_USER_3", "buildbot+dbt_test_user_3@dbtlabs.com"),
+}
+
+
+def env_variables(envs: dict[str, str]):
+    def env_variables_inner(ctr: dagger.Container):
+        for key, value in envs.items():
+            ctr = ctr.with_env_variable(key, value)
+        return ctr
+
+    return env_variables_inner
+
+
+def get_databricks_env_vars():
+    return {
+        "DBT_DATABRICKS_TOKEN": os.environ["DBT_DATABRICKS_TOKEN"],
+        "DBT_DATABRICKS_HOST_NAME": os.environ["DBT_DATABRICKS_HOST_NAME"],
+        "DBT_DATABRICKS_ENDPOINT": os.environ["DBT_DATABRICKS_ENDPOINT"],
+        "DBT_DATABRICKS_CLUSTER_NAME": os.environ["DBT_DATABRICKS_CLUSTER_NAME"],
+        "ODBC_DRIVER": "/opt/simba/spark/lib/64/libsparkodbc_sb64.so",
```

Comment:
Is there a reason not to define this in the workflow as well and import as an env var?